pkgs/development/libraries/science/biology/ciftilib/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, boost
+, libxmlxx
+, pkg-config
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ciftilib";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Washington-University";
+    repo = "CiftiLib";
+    rev = "v${version}";
+    sha256 = "xc2dpMse4SozYEV/w3rXCrh1LKpTThq5nHB2y5uAD0A=";
```

This hash format is barely used in derivations, better use the full SRI format.

```suggestion
    hash = "sha256-xc2dpMse4SozYEV/w3rXCrh1LKpTThq5nHB2y5uAD0A=";
```

---

Changed.